src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,33 @@
-export const TodoInfo = () => {};
+import React from 'react';
+import classNames from 'classnames';
+import { UserInfo } from '../UserInfo';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  todo: Todo;
+};
+
+export const TodoInfo: React.FC<Props> = ({ todo }) => {
+  const {
+    id,
+    title,
+    user,
+    completed,
+  } = todo;
+
+  return (
+    <article
+      data-id={id}
+      className={classNames(
+        'TodoInfo',
+        { 'TodoInfo--completed': completed },
+      )}
+    >
+      <h2 className="TodoInfo__title">
+        {title}
+      </h2>
+
+      <UserInfo user={user} />
```

Comment:
```suggestion
      {user && <UserInfo user={user} />}
```
Avoid unnecessary component rendering.